app/src/androidTest/java/net/skyscanner/backpack/compose/nudger/BpkNudgerTest.kt

```diff
@@ -76,4 +79,21 @@ class BpkNudgerTest : BpkSnapshotTest() {
             NudgerExample(initialValue = NudgerStoryMin - NudgerStoryMax)
         }
     }
+
+    @Test
+    fun defaultRow() = snap {
+        NudgerRowExample()
+    }
+
+    @Test
+    @Variants(BpkTestVariant.Default)
+    fun withNoSubtitle() = snap {
+        NudgerRowExample(subtitle = null)
+    }
+
+    @Test
+    @Variants(BpkTestVariant.Default)
+    fun withIcon() = snap {
```

Comment:
can you add one for without subtitle but with icon to ensure the alignment is correct across all variants?